package/libpiksi/libpiksi/src/util.c

```diff
@@ -667,6 +667,56 @@ int run_with_stdin_file2(const char *input_file,
   return status;
 }
 
+bool str_digits_only(const char *str)
+{
+  if (str == NULL) {
+    return false;
+  }
+
+  /* Empty string */
+  if (!(*str)) {
+    return false;
+  }
+
+  while (*str) {
+    if (isdigit(*str++) == 0) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+bool strtoul_all(int base, const char *str, unsigned long *value)
+{
+  char *endptr = NULL;
+  *value = strtoul(str, &endptr, base);
+  while (isspace(*endptr))
+    endptr++;
```

Comment:
Minor inconsistency with the brackets and oneliners in these `while` vs `if` statements.

Reply:
Fixed